README.md

```diff
@@ -31,16 +33,16 @@ import (
 
```

Comment:
verify that the example code runs properly on [playground](https://go.dev/play)

Reply:
https://go.dev/play/p/fYkdsj9V20U